metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/rest/DepublishedRecordView.java

```diff
@@ -12,28 +13,17 @@
 
   @JsonSerialize(using = ObjectIdSerializer.class)
   private ObjectId id;
-  private String recordId;
-  private DepublicationStatus depublicationStatus;
+  private final String recordId;
+  private final DepublicationStatus depublicationStatus;
```

Comment:
I thought about it, but then I wasn't sure. What was there before made total sense since it was mapping different values but now that is exactly the same values and it's an enum in this case shouldn't this suffice? 
Or you think we should have separate ones anyway? That could save us from having to change the type in a future change.. I'm in between those, so what's your preference?

Reply:
I usually favor having a separate one (a view vs a model) and I feel that should apply here too. But I will leave it up to you.

Reply:
Ok let's do that then.